gcs/reqtrace_bucket.go

```diff
@@ -29,6 +29,13 @@ type reqtraceBucket struct {
 	Wrapped Bucket
 }
 
+// GetWrappedWithReqtraceBucket returns the reference of reqtraceBucket which
+// wraps the given gcs.Bucket.
+func GetWrappedWithReqtraceBucket(b Bucket) (req *reqtraceBucket) {
+	req = &reqtraceBucket{Wrapped: b}
```

Comment:
don’t use naked returns, instead use `return &reqtraceBucket{Wrapped: b}` directly

Reply:
Oh, and also make the function signature read `func GetWrappedWithReqtraceBucket(b Bucket) *reqtraceBucket {`, no need to name that parameter.

Reply:
Nice! Thanks!